docs/data/material/components/dialogs/dialogs.md

```diff
@@ -168,8 +168,39 @@ The package [`material-ui-confirm`](https://github.com/jonatanklosko/material-ui
 
 Follow the [Modal accessibility section](/material-ui/react-modal/#accessibility).
 
-## Experimental APIs
-
-### Imperative API
-
-You can create and manipulate dialogs imperatively with the [`useDialog`](https://mui.com/toolpad/core/react-use-dialogs/) API in `@toolpad/core`. This API provides state management for opening and closing dialogs and for passing data to the dialog and back. It allows for stacking multiple dialogs. It also provides themed alternatives for `window.alert`, `window.confirm` and `window.prompt`.
+## Toolpad
+
+### `useDialogs`
+
+You can create and manipulate dialogs imperatively with the [`useDialogs`](https://mui.com/toolpad/core/react-use-dialogs/) API in `@toolpad/core`. This API provides state management for opening and closing dialogs and for passing data to the dialog and back. It allows for stacking multiple dialogs. It also provides themed alternatives for `window.alert`, `window.confirm` and `window.prompt`.
+
+The following example demonstrates how the `useDialogs` API can create a dialog that prompts the user to enter an ID and then confirms the deletion of an item with that ID. It waits for the async delete operation to complete before either displaying a success alert or a custom error dialog with an error payload:
```

Comment:
Is there a way to make it less verbose? As a developer, I would skip read the whole block because it's too long.